pyproject.toml

```diff
@@ -101,7 +101,7 @@ export = [
     "openvino>=2024.0.0", # OpenVINO export
     "tensorflow<=2.13.1; python_version <= '3.11'", # TF bug https://github.com/ultralytics/ultralytics/issues/5161
     "tensorflowjs>=3.9.0; python_version <= '3.11'", # TF.js export, automatically installs tensorflow
-    # "flatbuffers>=23.5.26,<100", # update old 'flatbuffers' included inside tensorflow package: WARNING Dockerfile error https://github.com/ultralytics/ultralytics/actions/runs/8715942435/job/23908971614
+    "flatbuffers>=23.5.26,<100", # update old 'flatbuffers' included inside tensorflow package
```

Comment:
@lakshanthad I had to comment this line out previously because it crashed the Dockerfile tests during docker deployments following PR merge. 

The Docker deployments can be run directly targeted to this branch using the Docker manual action and you can see the error message there.

Reply:
@lakshanthad ah actually I see you already ran the test and it appears to be working in https://github.com/ultralytics/ultralytics/actions/runs/8803839325, but this is actually not representative, I completely forgot that the Docker actions will pull from main branch rather than being able to specifically target a PR. The problem is here on L28, the Dockerfile clones `main` branch rather than the PR branch. 

We could test by temporarily changing this to target this branch. 

The reason I don't just copy the files into the Docker image is that this creates permission issues later if you try to i.e. `git pull` from within the Docker image.
https://github.com/ultralytics/ultralytics/blob/99f8e863ed1043b7c9a1cc9b68ead7830d4c2c57/docker/Dockerfile#L26-L28

Reply:
@glenn-jocher you are right. I changed Dockerfile in new branch and ran again with flatbuffers included and it fails in same place.
https://github.com/ultralytics/ultralytics/actions/runs/8870444509/job/24352237724

We might need to target flatbuffers specifically for `platform_machine == 'aarch64'` to resolve this.

Reply:
Ok let's try this!